.env

```diff
@@ -1,6 +1,7 @@
 _APP_ENV=development
 _APP_LOCALE=en
 _APP_WORKER_PER_CORE=6
+_APP_QUEUE_NAME=database_db_main
```

Comment:
Shouldn't this be empty by default?

Reply:
This is what will be set as the default `database` for a `project` in self-hosted, so that's the queue name that the database jobs will get pushed to as well

Reply:
Wouldn't this affect other workers as well?

Reply:
No, it will only affect containers that explicitly include it in the containers environment in the compose file

Reply:
We could hardcore it in the compose file for the database worker instead?

Reply:
OK, yeh this will make more sense to me